core/controllers/learner_group.py

```diff
@@ -485,13 +553,15 @@ class FacilitatorLearnerGroupViewHandler(base.BaseHandler):
 
     @acl_decorators.can_access_learner_groups
     def get(self, learner_group_id):
```

Comment:
Missing python typing for learner_group_id arg.

Reply:
Added